### ISSUE TITLE
Processing of 8KHz / 16KHz Sample rate audio signals

### ISSUE THREAD

**Issue by amadupu** [xiph/rnnoise#31]
though this question was posted earlier in the discussion threads, I could not see a definitive answer on the same. I would like to understand if the RNNoise Demo can be used for 8KHz / 16KHz PCM samples.

is the training depends on the sample rates?

can anyone suggest what changes I should perform in order to adapt the decoder to process 8KHz/16Khz sample rate speech

**Comment by venkat-kittu** [comment]
@amadupu I am also looking for same

**Comment by loretoparisi** [comment]
@amadupu I think the RNN must do train again, assumed the sample rate was different, or...we have to change the some hypeparameter otherwise?

**Comment by amadupu** [comment]
yeah..looks like retraining is required...but i feel there is more to it
than retrainng...esp with signal processing that needs to be done for lower
sampling rates..

@venkat-kittu try considering upsampling if that suits to your scope of work

regards
arun

**Comment by wegylexy** [comment]
If running at 8kHz can save some computing power, it would be significant when run in parallel.
